nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -330,7 +337,7 @@ in {
 
         passwordFile = lib.mkOption {
           type = lib.types.nullOr lib.types.path;
-          default = "/var/lib/mastodon/secrets/db-password";
+          default = null;
```

Comment:
I think this should be mentioned in release notes, as we change a default here.

Reply:
In this case a warning will appear:
```
error:
       Failed assertions:
       - <option>services.mastodon.database.passwordFile</option> needs to be set if
         <option>services.mastodon.database.createLocally</option> is not enabled.
(use '--show-trace' to show detailed location information)

```

Reply:
Right, makes sense! No need for release notes then.

Reply:
But people who have had a working setup before will have a broken one now without a simple means to fix it. They will not know what the previous location was and suddenly need to set it, it seems to me?